packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -53,9 +57,17 @@ class ComplianceRemediationButton extends React.Component {
         });
     }
 
-    noRemediationAvailable = () => {
+    remediationAvailable = () => {
         const { allSystems, selectedRules } = this.props;
```

Comment:
Providing some background here, in case you're unsure how the component works:

On system details, rules table:
`allSystems` always just contains the one system in question
`selectedRules` reflects what the user has selected in the rules table

On the policy details, systems table:
`allSystems` reflects what the user has selected in the systems table (misleading name; not _all_ systems, only selected)
`selectedRules` is always empty